File: Lee.EMS/8.1.0/aspnet-core/src/Lee.EMS.Core/BAS/DictionaryDetail.cs
using Abp.Domain.Entities.Auditing;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Lee.EMS.BAS
{
    [Table("Bas_DictionaryDetail")]
    public class DictionaryDetail : AuditedEntity<long>
    {
        [Column("DictionaryId")]
        [Required]
        public long DictionaryId { get; set; }

        [Column("Name")]
        [Required]
        [StringLength(150)]
        public string Name { get; set; }

        [Column("Value")]
        [Required]
        [StringLength(50)]
        public string Value { get; set; }

        [Column("Sort")]
        public int Sort { get; set; }

        [Column("Describe")]
        [StringLength(250)]
        public string Describe { get; set; }
        [Column("IsDefualt")]
        public bool IsDefualt { get; set; }
    }
}

File: Lee.EMS/8.1.0/vue/src/store/entities/dictionarydetail.ts

/*-------------------------------------------------------
// Copyright (C) 2020 Lic
//
// 功能描述:公共字典  实体对象
//
//
// 创建标识: Lee -- 2023-07-29 22:36:10
//
//------------------------------------------------------*/
 
import Entity from './entity'
// 公共字典 
export default class DictionaryDetail extends Entity<number>{
    dictionaryid:number;
    name:string;
    value:string;
    sort:number;
    describe:string;
    isDefualt:boolean;
}
